lesson25/script.js

```diff
@@ -0,0 +1,149 @@
+//1.Написать функцию, которая принимает 2 числа и возвращает -1, если первое меньше, чем второе; 1 – если первое больше, чем второе; и 0 – если числа равны.
+
+function returnNumbers(x, y) {
+  if (x < y) return -1;
+  else if (x > y) return 1;
+  else if (x == y) return 0;
+}
+
+//2.Написать функцию, которая вычисляет факториал переданного ей числа.
+
+function factorial(x) {
+  let sum = 1;
+  for (let i = 1; i <= x; i++) {
+    sum = sum * i;
+  }
+  return sum;
+}
+
+//3.Написать функцию, которая принимает три отдельные цифры и превращает их в одно число. Например: цифры 1, 4, 9 превратятся в число 149.
+
+function randomName(x) {
+  let sum = 1;
+  for (let i = 1; i <= x; i++) {
+    sum = sum * i;
+  }
+  return sum;
+}
+
+//Вариант 2 Нашел в интернете
+function randomName(a, b, c) {
+  let sum = a.toString() + b.toString() + c.toString();
+  return ++sum;
+}
+
+//4.Написать функцию, которая принимает длину и ширину прямоугольника и вычисляет его площадь. Если в функцию передали 1 параметр, то она вычисляет площадь квадрата.
+
+function randomName(x, y = x) {
+  let sum = x * y;
+  return sum;
+}
+
+//5.Написать функцию, которая проверяет, является ли переданное ей число совершенным. Совершенное число – это число, равное сумме всех своих собственных делителей.
+
+function randomName(x) {
+  let sum = 0;
+  for (let i = x - 1; i >= 0; i--) {
+    if (x % i == 0) sum += i;
```

Comment:
```suggestion
    if (x % i === 0) sum += i;
```
лучше везде использовать строгое равенство